.github/workflows/haskell.yml

```diff
@@ -110,20 +110,6 @@ jobs:
             cabal-version: 3.2
             cabal_project: cabal.project.user
 
-          - name: 8.8.4
-            ghc_version: 8.8.4
-            build: cabal-v2
-            runner: ubuntu-latest
-            cabal-version: 3.2
-            cabal_project: cabal.project.user
-
-          - name: 8.6.5
-            ghc_version: 8.6.5
-            build: cabal-v2
-            runner: ubuntu-latest
-            cabal-version: 3.2
-            cabal_project: cabal.project.user
-
```

Comment:
Why remove these?

Reply:
option "StandaloneKindSignatures" is not supported in these versions of ghc.

Reply:
this is an important package and we should support all versions of GHC that streamly supports. We do not need or use StandaloneKindSignatures .